lia-core/src/main/java/com/lithium/community/android/model/helpers/LiImage.java

```diff
@@ -85,36 +67,25 @@ public Long getWidth() {
         return width;
     }
 
-    public void setWidth(LiBaseModelImpl.LiInteger result) {
-        this.width = result.getValue();
+    public void setWidth(Long result) {
```

Comment:
Any reasons for moving from Int to Long?

Reply:
Although the name is Integer, it was holding a long value internally.